simtools/layout/layout_array.py

```diff
@@ -424,6 +489,7 @@ def read_telescope_list_file(self, telescope_list_file):
         self._initialize_corsika_telescope(table.meta)
         self._initialize_coordinate_systems(table.meta)
         self._load_telescope_list(table)
+        return table
```

Comment:
It seems you added this return of the table just for the tests. One of the tests (`test_include_radius_into_telescope_table`) doesn't seem necessary because the function `include_radius_into_telescope_table` is not actually used anywhere (and seems weird to me). The other test `test_plot_array` I think should be done by actually initialising the LayoutArray class instead and maybe adding a function to return the entire table after initialisation. 

I am not 100% sure about the last point since the telescope table in LayoutArray include all coordinates in it, but it's still probably good to do. We can discuss this point if you wish.

Reply:
if I remember correctly you agreed to return the table? 

this function is not used anywhere because we still dont have an application for plotting the array. it seems to me the function is necessary if you want to get a dictionary from a telescope file and then pass to the plot_array function. The ideia is to later create an application to do something similar to what we are doing in the test_plot_array. 

the LayoutArray instance is in any case used in the test_plot_array through 'layout_array_north_instance'.

I propose we keep it like this and if we dont like the application later on we can change both

Reply:
Yes, I agreed to keeping the part of returning the table. I think it's a good idea to have this option.